pkg/networkutils/network.go

```diff
@@ -358,8 +360,18 @@ func (n *linuxNetwork) SetupHostNetwork(vpcv4CIDRs []string, primaryMAC string,
 			return errors.Wrap(err, "ChangeLocalRulePriority: failed to delete priority 0 local rule")
 		}
 
-		// In IPv6 strict mode, ICMPv6 packets from the gateway must lookup in the local routing table so that branch interfaces can resolve their gateway.
+		// Add a high priority rule, to handle all link-local traffic by local table.
```

Comment:
Self Review: This is not the correct way to handle the link-local traffic. 
After adding this I realized that link-local traffic will not go through this route table. 

We need to use SGP "standard mode" for link local traffic.